src/region/RegionAnnotations.tsx

```diff
@@ -59,8 +59,22 @@ export default class RegionAnnotations extends React.Component<Props, State> {
         this.setStaged({ message: text });
     };
 
-    handleDraw = (shape: Rect): void => {
-        this.setStaged({ shape: this.scaleShape(shape, true) }); // Store at a scale of 1
+    handleDraw = (rawShape: Rect): void => {
+        const { staged } = this.props;
+        const newShape = this.scaleShape(rawShape, true);
+        const prevShape = staged && staged.shape;
+
+        if (
+            prevShape &&
```

Comment:
This is a little verbose. Another option would be to use `lodash/isEqual`, but we don't currently import it anywhere so it would add to the bundle size. Open to opinions.